src/app/compare/Compare.tsx

```diff
@@ -426,6 +430,8 @@ function getAllStats(comparisons: DimItem[], ratings: ReviewsState['ratings']) {
         stat.min = Math.min(stat.min, itemStat.value || 0);
         stat.max = Math.max(stat.max, itemStat.value || 0);
         stat.enabled = stat.min !== stat.max;
+        // lower # is better for drawtime and chargetime stats
+        stat.lowerBetter = [447667954, 2961396640].includes(itemStat.statHash);
```

Comment:
Nit: we could extract this into a shared function or constant (not a huge deal).